### DISCUSSION TITLE
New 2 Github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65536] [New to GitHub]
Thanks for taking the time to read this.
My name is CJ i am a programmer always learning.
Looking for work friends and connections.

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @ApexExpress 
 Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
